crash_similarity.py

```diff
@@ -15,10 +15,21 @@
 import download_data
 import utils
 
+import datetime
```

Comment:
You could import it like this: `from datetime import datetime, timedelta`. This way you can replace all `datetime.datetime.SOMETHING` in your code with `datetime.SOMETHING`.